### DISCUSSION TITLE
Welcome to openglobus-react Discussions!

### DISCUSSION THREAD

**Discussion by pavletto** [openglobus/openglobus-react#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Zemledelec** [comment]
It's great! I'd love to see the code examples right from the storybook, to not to go right into the code sources from the repo. I think it would be easier to get through the examples.
